test/unit/range/views/view_kmer_hash_test.cpp

```diff
@@ -17,12 +19,31 @@
 #include <seqan3/range/views/take_until.hpp>
 #include <seqan3/range/views/to.hpp>
 
-#include <gtest/gtest.h>
+#include "../iterator_test_template.hpp"
 
 using seqan3::operator""_dna4;
 using seqan3::operator""_dna5;
 using seqan3::operator""_shape;
 
+using iterator_type = std::ranges::iterator_t< decltype(std::declval<seqan3::dna4_vector&>()
```

Comment:
remove this whitespace? 💅 
```suggestion
using iterator_type = std::ranges::iterator_t<decltype(std::declval<seqan3::dna4_vector&>()
```